### PR TITLE
chore: update featured list

### PR DESCRIPTION
**Pull request by jcountsNR**
Adding and removing a few IO listings from featured list.

### CODE REVIEW

File: quickstarts/codestream/codestream/config.yml [hunk 1]
```diff
@@ -40,4 +40,3 @@ documentation:
 website: https://codestream.com
 # Keywords for filtering / searching criteria in the UI
 keywords:
```
Comment (brammerl) [inline quickstarts/codestream/codestream/config.yml:42]: If there aren't any keywords I think you may have to delete the field all together.

Comment (louisleungk) [reply quickstarts/codestream/codestream/config.yml:42]: maybe we can add a tag for "IDE"?

**Comment by brammerl** [comment]
@louisleungk requested we add `featured` keyword to docker otel quickstart so i went ahead and did that.

**Comment by seemantk** [comment]
Thank you @brammerl

**Review by seemantk — DISMISSED** [review]

File: quickstarts/codestream/codestream/config.yml (hunk 1 above)
Comment (brammerl) [reply quickstarts/codestream/codestream/config.yml]: @seemantk what do you think of this suggestion? Seems fair to me.

**Review by brammerl — APPROVED** [review]

**Pull request merged**